schemas/system/domains.schema

```diff
@@ -0,0 +1,22 @@
+{
+  IdStatus: Enum('Prealloc', 'Init', 'Actual', 'Historical'),
+  StorageKind: Enum('Master', 'Cache', 'Backup', 'Replica'),
+  CategoryRealm: Enum('Global', 'System', 'Local'),
+  CategoryFamily: Enum('Registry', 'Dictionary', 'Table', 'Log', 'View'),
+  UserAccess: Bitmask('Read', 'Insert', 'Update', 'Delete', 'Execute', 'Audit'),
+  Compression: Enum('None', 'GZIP', 'ZIP'),
+  SessionToken: { type: 'string', length: 64 },
+  MimeType: Enum(
```

Comment:
Also, I'd suggest to either use `MIMEtype` or `Mimetype` or even `MediaType` (as mime type is an old name). (Refs https://en.wikipedia.org/wiki/Media_type and https://www.iana.org/assignments/media-types/media-types.xhtml)

Reply:
MediaType is fine